topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -400,9 +400,8 @@ The BAM file contains information for all our reads, making it difficult to insp
 > 1. Install [**IGV**](https://software.broadinstitute.org/software/igv/download) (if not already installed)
 > 2. Start IGV locally
 > 3. Click on the collection `RNA STAR on collection N: mapped.bam` (output of **RNA STAR** {% icon tool %})
-> 4. Expand the {% icon param-file %} `GSM461177_untreat_paired` file.
```

Comment:
could you please keep this instruction, it's still required to expand before visualising. Otherwise the lines you've added below look good.

Reply:
edited

Reply:
Sorry @hrukkudyr could you double check? I'm reviewing this PR again but I still see the "Expand the ..." line as deleted by this PR.